src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -596,6 +596,14 @@ static constexpr Property<ExecutionMode> execution_mode{"EXECUTION_MODE_HINT"};
 static constexpr Property<uint64_t, PropertyMutability::RW> dynamic_quantization_group_size{
     "DYNAMIC_QUANTIZATION_GROUP_SIZE"};
 
+/**
+ * @brief This property defines unsupported group size for dynamic quantization optimization
+ * @ingroup ov_runtime_cpp_prop_api
+ *
+ * If dynamic_quantization_group_size is larger than this unsupported value, dynamic quantization will be disabled.
+ */
+static constexpr Property<uint64_t, PropertyMutability::RW> dynamic_quantization_group_size_unsupported{
```

Comment:
would be `max_dynamic_quantization_group_size` a better name?

Reply:
updated, thanks!